tests/gis_tests/gdal_tests/test_srs.py

```diff
@@ -279,3 +285,17 @@ def test_unicode(self):
             self.assertEqual(srs.wkt, wkt)
             self.assertIn('Langschoß', srs.pretty_wkt)
             self.assertIn('Langschoß', srs.xml)
+
+    def test_axis_ordering(self):
+        if GDAL_VERSION < (3, 0):
+            self.skipTest("GDAL >= 3.0 is required for this test.")
```

Comment:
Why not `@unittest.skipIf(GDAL_VERSION < (3, 0), 'GDAL >= 3.0 is required for this test.')`?